runtime/src/accounts_db.rs

```diff
@@ -5234,21 +5238,15 @@ impl AccountsDb {
         )>,
     ) -> Result<(Hash, u64), BankHashVerificationError> {
         let mut scan_and_hash = move || {
-            // # of passes should be a function of the total # of accounts that are active.
-            // higher passes = slower total time, lower dynamic memory usage
-            // lower passes = faster total time, higher dynamic memory usage
-            // passes=2 cuts dynamic memory usage in approximately half.
-            let num_scan_passes: usize = 2;
-
-            let bins_per_pass = PUBKEY_BINS_FOR_CALCULATING_HASHES / num_scan_passes;
+            let bins_per_pass = PUBKEY_BINS_FOR_CALCULATING_HASHES / NUM_SCAN_PASSES;
```

Comment:
Why not just use `BINS_PER_PASS` here?

Reply:
sure 'nuff. this used to fail in rust const eval, I thought.